NEWS.txt

```diff
@@ -9,7 +9,7 @@ Version 7.0 - not yet released
   - parse wind from standard NMEA sentence WMV
   - driver for XC Tracer Vario
   - driver for KRT2 radio
-  - driver for Air Control Display altimeter
+  - driver for Air Control Display altimeter and radio
```

Comment:
This should be part of the commit that adds the feature.

Reply:
Radio driver would be complete with the addition of this PR. Will squash this commit into the "main" one.